containers/gcp/content/run-with-gce.sh

```diff
@@ -107,45 +106,21 @@ else
   INSTANCE_PREFIX="datalab-${ESCAPED_USER_EMAIL}"
 fi
 
-INSTANCE=`gcloud compute instances list --project "${PROJECT}" --zone "${ZONE}" --regex "${INSTANCE_PREFIX}-[0-9]*" --limit 1 --format "value(name)"`
+INSTANCE=`gcloud compute instances list --project "${PROJECT_ID}" --zone "${ZONE}" --regex "${INSTANCE_PREFIX}-[0-9]*" --limit 1 --format "value(name)"`
```

Comment:
I tried to find out how auth works here but I couldn't...On container startup it is not authenticated, right? How does "gcloud compute instances" work? Maybe you can add some comments here to explain it?